module/VuFind/src/VuFind/Service/MarkdownFactory.php

```diff
@@ -172,11 +185,13 @@ protected function getBaseConfig(): array
      */
     protected function getExtensionClass(string $extension): string
     {
-        $extensionClass = sprintf(
-            'League\CommonMark\Extension\%s\%sExtension',
-            $extension,
-            $extension
-        );
+        $extensionClass = (strpos($extension, '\\') !== false)
```

Comment:
Would it make sense to use a `class_exists` here, or do you want to support service names that don't necessarily match existing class names?

Reply:
The `class_exists` is used few lines later for checking both - custom and commonmark internal extensions.

I did not want to support non-class service names - but it could be done in future if needed.

Reply:
Okay, makes sense! I agree that there is probably not a strong reason to support non-class service names.